public/pages/ReviewAndCreate/components/FilterDisplayList/FilterDisplayList.tsx

```diff
@@ -30,7 +30,7 @@ export const FilterDisplayList = (props: FilterDisplayListProps) => {
 
   // Old detectors with custom filters will have no filter list, but
   // will have a populated filter query
-  if (isEmpty(filters) && isOldDetector && !isEmpty(props.filterQuery)) {
```

Comment:
Looks like you can actually remove the `isOldDetector` check entirely. If it's true, it's from an old detector that's been created/updated on old UI. If it's false, it's only been created/updated from API (no `uiMetadata.*` fields). Regardless of either of those situations, we want to show the custom filter if `props.filterQuery` isn't empty.